### PR TITLE
test(useIsMounted): add positive and negative test

### PR DESCRIPTION
**Pull request by DidrikLind**
Adding two basic tests for isMounted. I am a bit new with testing of hooks and saw this as an opportunity to learn and contribute.

Hope it will come in handy!

Cheers

### CODE REVIEW

**Comment by juliencrn** [comment]
Hi @DidrikLind, thanks for your contribution, looks good! I will review it next week.

**Comment by DidrikLind** [comment]
@juliencrn sounds good.

Happy new year 🎉

**Comment by juliencrn** [comment]
It's perfect! You can omit repeating the hook name in tests because it's already scoped in the `describe()`.

File: lib/src/useIsMounted/useIsMounted.test.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { renderHook } from '@testing-library/react-hooks/native'
+
+import useIsMounted from './useIsMounted'
+
+describe('useIsMounted()', () => {
+  test('isMounted() should return true when component is mounted', () => {
```
Comment (juliencrn) [inline lib/src/useIsMounted/useIsMounted.test.ts:6]:
```suggestion
  test('should return true when component is mounted', () => {
```

File: lib/src/useIsMounted/useIsMounted.test.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    const {
+      result: { current: isMounted },
+    } = renderHook(() => useIsMounted())
+
+    expect(isMounted()).toBe(true)
+  })
+
+  test('isMounted() should return false when component is unmounted', () => {
```
Comment (juliencrn) [inline lib/src/useIsMounted/useIsMounted.test.ts:14]:
```suggestion
  test('should return false when component is unmounted', () => {
```

**Review by juliencrn — APPROVED** [review]
Thanks again 👍

**Pull request merged**